co-simulation/lib/mosaic-common-utils/src/main/java/org/eclipse/mosaic/lib/CommonUtil/ambassador/CommonInstanceManager.java

```diff
@@ -40,7 +40,7 @@
 
 
 public class CommonInstanceManager<T extends CommonInstance, R extends CommonRegistrationMessage> {
-    protected  Map<String, T> managedInstances = new HashMap<>();
+    public  Map<String, T> managedInstances = new HashMap<>();
```

Comment:
What's the rationale for making this public as opposed to adding a `getInstances()` accessor method or something like that?